tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -327,7 +347,7 @@ public function userUpdatesLastPublicLinkShareUsingTheGraphApiWith(string $user,
 			$spaceId,
 			$itemId,
 			\json_encode($body),
-			$this->featureContext->shareNgGetLastCreatedLinkShareID()
+			$permissionID
 		);
 		$this->featureContext->setResponse($response);
```

Comment:
setResponse in `When` step method only
```suggestion
		return $response;
```